utils/scapp.h

```diff
@@ -604,12 +698,37 @@ scApp::processOption(argparser& parser, int opt)
         break;
       case 1010:
         options.bopts.verbose = true;
+		options.astcopts.verbose = true;
         capture = false;
         break;
       case 1011:
         options.bopts.noSSE = true;
         capture = true;
         break;
+	  case 1012: // astc_blk_s
+		options.astcopts.blockSize = getAstcBlockSize(parser.optarg.c_str());
+		hasArg = true;
+		capture = false;
+		break;
+	  case 1013: // astc_func
+		options.astcopts.function = getAstcEncoderFunction(parser.optarg.c_str());
+		hasArg = true;
+		capture = false;
+		break;
+	  case 1014: // astc_mode
+		options.astcopts.mode = getAstcEncoderMode(parser.optarg.c_str());
+		hasArg = true;
+		capture = false;
+		break;
+	  case 1015: // astc_quality
+		options.astcopts.qualityLevel = getAstcQualityLevel(parser.optarg.c_str());
+		hasArg = true;
+		capture = false;
+		break;
+	  case 1016: // astc_normal
+		options.astcopts.normalMap = true;
+		capture = false;
+		break;
```

Comment:
Wrong indentation on this and 23 preceding lines.

Reply:
Fixed same as above